compiler/internal/cuegen/testdata/basic_config_svc.cue

```diff
@@ -14,11 +14,11 @@ package svc
 // referenced from other config values for example when configuring a callback URL:
 //    CallbackURL: "\(#Meta.APIBaseURL)/webhooks.Handle`"
 #Meta: {
-	APIBaseURL: string @tag(ApiURL) // The base URL which can be used to call the API of this running application.
+	APIBaseURL: string @tag(APIBaseURL) // The base URL which can be used to call the API of this running application.
```

Comment:
Note: this is just re-generating the golden files from a previous PR's change, so these `*.cue` file changes are un-related